src/index.html

```diff
@@ -4,9 +4,197 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="navigation">
+      <a class="navigation__logo" href="#">
+        <img src="images/logo.png" alt="logo moyo">
+      </a>
+      <nav>
+        <ul class="navigation__list">
+          <li class="navigation__item" ><a href="#" class="navigation__item__link is-active">Apple</a></li>
+          <li class="navigation__item"><a href="#" class="navigation__item__link">Samsung</a></li>
+          <li class="navigation__item"><a href="#" class="navigation__item__link">Смартфоны и телефоны</a></li>
+          <li class="navigation__item"><a href="#" class="navigation__item__link hover" data-qa="nav-hover">Ноутбуки и компьютеры</a></li>
+          <li class="navigation__item"><a href="#" class="navigation__item__link">Гаджеты</a></li>
+          <li class="navigation__item"><a href="#" class="navigation__item__link">Планшеты</a></li>
+          <li class="navigation__item"><a href="#" class="navigation__item__link">Фото</a></li>
+          <li class="navigation__item"><a href="#" class="navigation__item__link">Видео</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="row__product-cards">
+      <div class="product-card" data-qa="card">
+        <img src="images/imac.jpeg" alt="monitor" class="product-card__photo">
```

Comment:
аймак это не монитор, это целая система
```suggestion
        <img src="images/imac.jpeg" alt="imac photo" class="product-card__photo">
```

Reply:
но на фотке же аймак манитор))
@danheim

Reply:
fixed)
нравится фиксить штуки на которые тратишь меньше минуты))